pulpanalytics/tests/test_recieve_telemetry.py

```diff
@@ -2,8 +2,8 @@
 from django.urls import reverse
```

Comment:
You probably also want to rename this file.

Reply:
Yup. Fixing it now, thanks.